tests/syscall/fs/host/host.c

```diff
@@ -78,26 +88,105 @@ int wmain(int argc, wchar_t* argv[])
     // Windows does not support umask.
     // Please set up the right permission to the parent directory.
 
-    int ret = test_fs_posix(enclave_path, src_dir, tmp_dir);
+    rmdir(argv[3]);
 
-    free(src_dir);
-    free(tmp_dir);
+    r = oe_create_fs_enclave(enclave_path, type, flags, NULL, 0, &enclave);
+    OE_TEST(r == OE_OK);
 
-    return ret;
+    r = test_fs(enclave, src_dir, tmp_dir);
+    OE_TEST(r == OE_OK);
+
+    r = oe_terminate_enclave(enclave);
+    OE_TEST(r == OE_OK);
+
+    printf("=== passed all tests (hostfs)\n");
+
+    return 0;
 }
+#else
+int rmdir(const char* path)
```

Comment:
I would name this something else to avoid confusion with the linux syscall name

Reply:
Fixed, thanks!